demo/docker-compose.yml

```diff
@@ -78,6 +78,10 @@ services:
     networks:
       - backend
       - frontend
+    healthcheck:
+      start_period: 30s
```

Comment:
are you sure 30s is enough for the api-server to be ready? The api being used in mf2c is much older than this one, so I don't know if the initialization times have changed...

Reply:
We will figure this out during out tests.